test/runTest.sh

```diff
@@ -21,8 +37,17 @@ for name in $TESTS; do
   echo $name
   echo ""
   psql temporal_tables_test -X -a -q --set=SHOW_CONTEXT=never < test/sql/$name.sql > test/result/$name.out 2>&1
-  diff -b test/expected/$name.out test/result/$name.out
-done
+  DIFF_OUTPUT=$(diff -b test/expected/$name.out test/result/$name.out)
 
+  if [ -n "$DIFF_OUTPUT" ]; then
+    # Expected and actual files are different.
+    FILES_DIFFERENT=true
+  fi
```

Comment:
The reason behind introducing FILES_DIFFERENT flag was to return exit code 1 whenever there is a failure. This will make sure that CI pipeline checks will report failure when any of the test case fails.
Prior to this PR, CI pipeline checks would still pass even when there are test case failures, the reason being these scripts return exit code 0 in all cases.
In think, the only part missing in the new code is, printing of the diff. I will make that quick change now.
Let me know if this sounds good.

Reply:
ok, yes I wasn't sure but your explanation makes sense